2023-04-19 23:32:52,327 - root - ERROR - ws_client.py - __init__ : 27 - device robot ws://192.168.1.200:4421 down
2023-04-19 23:32:55,382 - root - ERROR - ws_client.py - __init__ : 27 - device sceno ws://192.168.1.220:4421 down
